src/main/java/org/openelisglobal/login/bean/UserSession.java

```diff
@@ -6,7 +6,12 @@
 
 public class UserSession {
 
+    public enum LoginMethod {
+        FORM, SAML, OAUTH, CERT, NONE
```

Comment:
can we rename `FORM ` to `BASIC` to make it sound generic for both the REST API and the UI Login ? i guess its the same Login mechanism for both the  REST API calls and the UI

Reply:
Ah, Basic should be added, but is distinct from form authentication